daemon/daemon_unix.go

```diff
@@ -241,6 +271,16 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 		logrus.Warnf("Your kernel does not support Block I/O weight_device. Weight-device discarded.")
 		hostConfig.BlkioWeightDevice = []*pblkiodev.WeightDevice{}
 	}
+	if len(hostConfig.BlkioReadBpsDevice) > 0 && !sysInfo.BlkioReadBpsDevice {
+		warnings = append(warnings, "Your kernel does not support Block read limit in bytes per second.")
```

Comment:
This error message isn't clear --- is the flag not supported or simply the configuration the user provided? If the flag is not supported period because of the type of device. Hard to tell from the language. Some better choices that give users some better choices.

```
Your kernel does not support the --read-bps-device flag .
Your kernel does not support the --read-bps-device flag with the XXX device. 
You specified the --read-bps-device=xXXX:XXX values. Your kernel does not support this configuration. Specify another.
```